AlicerceTests/Logging/FileProviderTests.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  FileProviderTests.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 04/04/17.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import XCTest
+@testable import Alicerce
+
+class FileProviderTests: XCTestCase {
+    
+    var documentsPath: String?
+    var logfileURL: URL?
+    var provider: FileProvider?
+    
+    override func setUp() {
+        super.setUp()
+        
+        self.documentsPath = "file:///tmp/Log.log"
+        self.logfileURL = URL(string: self.documentsPath!)!
+        self.provider = FileProvider(fileURL: self.logfileURL!)
+        self.provider!.clear()
+        self.provider!.minLevel = .error
+        self.provider!.formatter = LogItemStringFormatter(formatString: "$M")
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+        Log.removeAllProviders()
```

Comment:
You should also set the class variables to `nil` here to free resources sooner 😉